### DISCUSSION TITLE
Displayless tulip

### DISCUSSION THREAD

**Discussion by BadPotato1007** [shorepine/tulipcc#287] [Ideas]
Hey! So I was just thinking... The display is the most expensive part, right? So maybe, someone could help write a version that requires just the board, that pretty simple to make, and the tulip can either host a site on an access point, or maybe make an app that communicates with the tulip via bluetooth or web sockets? That way people can even use bigger screens, or simply use their phones or ipads for this!

**Comment by bwhitman** [comment]
You're in luck! 
Check out this thread https://github.com/shorepine/tulipcc/discussions/115
